src/android/build.gradle

```diff
@@ -7,8 +7,8 @@ buildscript {
         compileSdkVersion = 34
         targetSdkVersion = 34
 
-        ndkVersion = "26.2.11394342"
-        kotlinVersion = "1.8.0"
+        ndkVersion = "26.1.10909125"
```

Comment:
why did you downgrade it? put back to 26.2.11394342

Reply:
I followed the react native upgrade helper:
https://react-native-community.github.io/upgrade-helper/?from=0.73.8&to=0.75.2